.github/workflows/cicd.yml

```diff
@@ -0,0 +1,60 @@
+name: CICD
+
+on:
+  pull_request:
+    types: [opened, synchronize]
```

Comment:
I think it might make more sense to have the action deploy when there are pushes to main and then just weekly after the update action has executed.

Reply:
Good point, just removed it

Reply:
I think we should also add this trigger:
```
schedule:
    # Run Mondays at 5:15 am
    - cron: "15 5 * * 0"
```

Reply:
Done!